### PR TITLE
btcpayserver: 1.5.1 -> 1.5.3

### PR DESCRIPTION
**Pull request by erikarvstedt**
- [x] Tested with the [nix-bitcoin VM test](https://gist.github.com/60c8e9f94ece2e5432d9f7ccaf53cd6a).

Notes for reviewers:
You can verify GPG signatures with `refetch=1 pkgs/applications/blockchains/btcpayserver/update.sh`.

cc @prusnak @nixbitcoin

### CODE REVIEW

**Comment by prusnak** [comment]
This PR is missing extra changes in `pkgs/applications/blockchains/nbxplorer/util/update-common.sh` from https://github.com/NixOS/nixpkgs/pull/173377/commits/6d50e9554a2c04d8c4666e9290297625d0c0144e#diff-1e95ad7af6f5dd0c040b43f049de59e8ce1841bbe5af8c9eab31c2263919e879

Is this intentional?

**Comment by erikarvstedt** [comment]
It seems that ndorier is back at signing releases, so we don't need to add kukk's key for now.
But you're right, the error msg trap is useful in general, so I've put it back in again.

File: pkgs/applications/blockchains/nbxplorer/util/update-common.sh [hunk 1]
```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p coreutils curl jq common-updater-scripts dotnet-sdk_3 git gnupg nixFlakes
+#!nix-shell -i bash -p coreutils curl jq common-updater-scripts dotnet-sdk_6 git gnupg nixFlakes
 set -euo pipefail
 
 # This script uses the following env vars:
 # getVersionFromTags
 # refetch
 
+trap 'echo "Error at ${BASH_SOURCE[0]}:$LINENO"' ERRd
```
Comment (prusnak) [inline pkgs/applications/blockchains/nbxplorer/util/update-common.sh:9]:
```suggestion
trap 'echo "Error at ${BASH_SOURCE[0]}:$LINENO"' ERR
```

**Comment by prusnak** [comment]
@GrahamcOfBorg build btcpayserver nbxplorer

File: pkgs/applications/blockchains/nbxplorer/util/update-common.sh (hunk 1 above)
Comment (erikarvstedt) [reply pkgs/applications/blockchains/nbxplorer/util/update-common.sh]: Pheeew, thank you :smiley_cat:

**Pull request merged**